Makefile

```diff
@@ -18,13 +18,20 @@
 # BOLOS_SDK IS  DEFINED	 	We use the plain Makefile for Ledger
 # BOLOS_SDK NOT DEFINED		We use a containerized build approach
 
-#TESTS_JS_PACKAGE = "@zondax/ledger-algorand"
```

Comment:
I think if you uncomment these lines you won't need to call the `cd js` you have on line 46